models/issue_assignees.go

```diff
@@ -0,0 +1,264 @@
+// Copyright 2018 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"fmt"
+
+	"code.gitea.io/gitea/modules/log"
+	api "code.gitea.io/sdk/gitea"
```

Comment:
`code.gitea.io/sdk` is the external package. Please move to end of import section.

Reply:
Done.